### PR TITLE
fix(config): update Philio PSM02 Configuration

### PR DESCRIPTION
**Pull request by orandasoft**
- Add lifeline association group
- Update configuration parameter 5 and 6 to use partial parameters
- Add configuration parameter 7 as partial parameters
- Some minor cosmetic changes

### CODE REVIEW

File: packages/config/config/devices/0x013c/psm02.json [hunk 1]
```diff
@@ -14,10 +14,12 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"supportsZWavePlus": false,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x013c/psm02.json]:
```suggestion
```
We don't use that anymore

File: packages/config/config/devices/0x013c/psm02.json [hunk 2]
```diff
@@ -46,19 +48,311 @@
...
+					"value": 1
+				},
+				{
+					"label": "Enable",
+					"value": 0
+				}
+			]
+		},
+		"7[0x02]": {
+			"label": "Send motion OFF report. ",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x013c/psm02.json]:
```suggestion
			"label": "Send Motion Off Report",
```
Please check that all other labels are *Title Case* without punctuation.

File: packages/config/config/devices/0x013c/psm02.json [hunk 3]
```diff
@@ -103,4 +397,9 @@
 			"defaultValue": 12
 		}
 	}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x013c/psm02.json]:
```suggestion
	},
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x013c/psm02.json (hunk 1 above)
Comment (orandasoft) [reply packages/config/config/devices/0x013c/psm02.json]: Found that in the PST02A configuration.

**Comment by orandasoft** [comment]
Corrected case in one additional line.

**Comment by orandasoft** [comment]
How do I run prettier?

**Comment by AlCalzone** [comment]
If you're using VSCode and have installed the recommended extensions and all dependencies, you just need to save the file. Otherwise:
@zwave-js-bot fix lint

**Comment by orandasoft** [comment]
This newbie will use @zwave-js-bot fix lint until graduating to VSCode.

**Comment by orandasoft** [comment]
Is there anything left to do?

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
Just noticed a little thing you didn't touch. Changes look good otherwise.

**Pull request merged**